openedx/core/djangoapps/schedules/tests/test_resolvers.py

```diff
@@ -248,7 +248,7 @@ def create_resolver(self, user_start_date_offset=8):
     def test_schedule_context(self):
         resolver = self.create_resolver()
         # using this to make sure the select_related stays intact
-        with self.assertNumQueries(15):
+        with self.assertNumQueries(26):
```

Comment:
Ugh. I tried to understand why the jump here was so much, but I don't understand it. It's all checking the flag, but I couldn't find where/why it is checking it so often.